kornia/geometry/conversions.py

```diff
@@ -1175,3 +1185,163 @@ def normalize_homography3d(
     # compute chain transformations
     dst_norm_trans_src_norm: torch.Tensor = dst_norm_trans_dst_pix @ (dst_pix_trans_src_pix @ src_pix_trans_src_norm)
     return dst_norm_trans_src_norm
+
+
+def extrinsics_from_Rt(R: Tensor, t: Tensor) -> Tensor:
+    r"""Combines 3x3 R and 1x3 t into 4x4 extrinsics.
+    Extrinsics matrix is usually named `K`, but we avoid this,
+    to not interfere with `import kornia as K`
+
+    Args:
+        R: Rotation matrix, :math:`(B, 3, 3).`
+        t: Translation matrix :math:`(B, 3, 1)`.
+
+    Returns:
+        the extrinsics :math:`(B, 4, 4)`.
+
+    """
+    KORNIA_CHECK_SHAPE(R, ["B", "3", "3"])
+    KORNIA_CHECK_SHAPE(t, ["B", "3", "1"])
+    Rt = torch.cat([R, t], dim=2)
+    return convert_affinematrix_to_homography3d(Rt)
```

Comment:
the name of the funcion `convert_affinematrix_to_homography3d(Rt)` is misleading. This function is baswically making homogeneous a matrix, if that can be said in this way.

Reply:
Yes, but it is not a PR which added that function ;) we had `convert_affinematrix_to_homography3d` for ages. 
And I believe that the name is OK, because [2x3] is A, but [2x3; 0 0 1] is H.

Reply:
but here is 3x4 (not 2x3) and the concept of 3d homography is weird to me. But yes, I know it is an old function.